plugins/virtual_event/services/VirtualEventService.php

```diff
@@ -89,10 +102,20 @@ public function updateAction($id, KalturaVirtualEvent $virtualEvent)
 		$this->validateScheduleEvents($virtualEvent);
 		$this->validateGroups($virtualEvent);
 		
-		// save the object
-		/** @var VirtualEvent $dbVirtualEvent */
-		$dbVirtualEvent = $virtualEvent->toUpdatableObject($dbVirtualEvent);
-		$dbVirtualEvent->save();
+		try
+		{
+			// save the object
+			/** @var VirtualEvent $dbVirtualEvent */
+			$dbVirtualEvent = $virtualEvent->toUpdatableObject($dbVirtualEvent);
```

Comment:
Same comment as above 
Validate the json string is valid in validateForUpdate if it was sent on the virtualEvent object.